webclient/public/locales/es/translation.json'

```diff
@@ -1 +1,382 @@
\ No newline at end of file
+{
+  "Common": {
+    "language": "Inglés",
```

Comment:
FYI, people did translate this string labeling the language, instead putting the target language (their own language). I fixed this for the languages that already have some translations for the webclient. It now shows the same as in the Cockatrice app.

The string is now also locked to not allow any more translations.
However, this change is not present in GitHub yet.